app/src/test/java/com/codesoom/assignment/application/UserServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@DisplayName("UserService 는")
+class UserServiceTest {
+    @Autowired
+    UserService userService;
+    @Autowired
+    UserRepository userRepository;
+    private final User userWithoutId = new User(null, "김갑생", "gabseng@naver.com", "12341234");
+
+    @BeforeEach
+    void setUp() {
+        userRepository.deleteAll();
+    }
+
+    @Nested
+    @DisplayName("create() 메서드에서")
```

Comment:
메소드를 "장소"처럼 묘사하지 말고 행동하는 대상으로 생각하고 작성해 보세요.
```suggestion
    @DisplayName("create() 메서드는")
```

Reply:
네! 알겠습니다!